packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -49,16 +51,18 @@ export const RadioButton = compose<IRadioButtonType>({
     // 1) Calls pressable's onFocus in order to keep track of our state's focus variable. It is dependent on pressable's
     //    focus variable. Without this, it wouldn't stay updated because we're overriding it's onFocus below for the rootProps.
     // 2) Selects the currently focused button by calling the RadioGroup's callback function.
-    const onFocusChange = React.useCallback((/* ev: NativeSyntheticEvent<{}> */) => {
-      // This check is necessary because this func gets called even when a button loses focus (not sure why?) which then calls the client's onChange multiple times
+    const onFocusChange = React.useCallback((ev: NativeSyntheticEvent<{}>) => {
```

Comment:
is `NativeSyntheticEvent<{}>` really the right type here?  The cast to `FocusEvent` signals to me that you should just use `FocusEvent` throughout. This is one of those weird things where we can't really make guarantees on the type with absolute certainty (maybe turbomodules with type generation in the future and the corresponding viewmanager changes make this safer).

Reply:
The onFocus callback on ViewWin32.Props is: `onFocus?: (ev: RN.NativeSyntheticEvent<{}>) => void;`